.php_cs.dist

```diff
@@ -121,6 +120,33 @@ return Config::create()
         'strict_comparison' => true,
         'yoda_style' => false,
         'single_line_throw' => false,
+        'compact_nullable_typehint' => true,
```

Comment:
What do you think about keeping rules sorted by name?

Reply:
:+1